ui/src/routes/Activities/Activities.js

```diff
@@ -117,18 +62,21 @@ class Subjects extends Component {
 
     let epubUrl = `/static/content_player/index.html?epubUrl=/content/${this.props.subjectName}/${this.props.gradeName}/${this.props.unitName}/${this.props.lessonName}`
     return (
-      <div style={styles.container} >
-        <header style={styles.header}>
-          <h1 className="logo">CLIx Connected Learning Initiative</h1>
-          <p className="unplat-v">unplatform version {this.props.version}</p>
-          <nav style={styles.nav}>
-            <button onClick={this._onChooseTool}>{this.props.strings.unplatformNav.chooseTool}</button>
-            <button onClick={this._onChooseLesson}>{this.props.strings.unplatformNav.chooseNewLesson}</button>
-            <button onClick={this._onToggleModal}>{this.props.strings.unplatformNav.finishLesson}</button>
+      <div className='act-container'>
+        <header className='c-header'>
+          <h1 className='c-header__logo' aria-label='Clix Connected Learning Initiative'>CLIx Connected Learning Initiative</h1>
+          <p className='c-header--unplat-v'>unplatform version {this.props.version}</p>
+          <nav className='c-header__nav'>
+            <button onClick={this._onChooseTool}>
+              {this.props.strings.unplatformNav.chooseTool}</button>
+            <button onClick={this._onChooseLesson}>
+              {this.props.strings.unplatformNav.chooseNewLesson}</button>
+            <button onClick={this._onToggleModal}>
+              {this.props.strings.unplatformNav.finishLesson}</button>
           </nav>
         </header>
-        <main className="span_12_of_12" style={styles.background}>
-          <iframe src={epubUrl} className="fill-win" frameBorder="0" allowfullscreen></iframe>
+        <main className='span_12_of_12'>
+          <iframe src={epubUrl} className='act-iframe--fill-win' frameBorder='0' allowFullScreen />
```

Comment:
Should this `iframe` have a `title` attribute? I must have forgotten about it in the other PRs -- do you want to add one here, or I can open another PR...

Reply:
"A couple of tweaks" means the minor css tweaks that are related to accessibility improvements. I'll be working on those along with the iframe title fix in an accessibility-related feature branch, so we can better track those improvements in the git history.

That means we can merge this in and make the fixes I described above on a new feature branch.